tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -5088,16 +5085,16 @@ def get_matmul(arr):
     integer32 = randint(min_int32, max_int32, size=size, dtype=np.int32)
     integer64 = randint(min_int64, max_int64, size=size, dtype=np.int64)
 
-    fl = uniform(min_float / 2, max_float / 2, size = size)
-    fl32 = uniform(min_float32 / 2, max_float32 / 2, size = size)
+    fl = uniform(min_float / size, max_float / size, size = size)
```

Comment:
The min/max should be:
- min : `-np.sqrt(-min_float / size)`
- min : `np.sqrt(max_float / size)`

remember `(a,b,c) @ (d,e,f) = a*d + b*e + c*f` so `max_float_result = (max_float_element**2)*size`